nlptest/transform/robustness.py

```diff
@@ -59,10 +59,14 @@ async def run(sample_list: List[Sample], model: ModelFactory, **kwargs) -> List[
                 if 'original_context' in sample.__annotations__:
                     dataset_name = sample.dataset_name.split('-')[0].lower()
                     user_prompt = kwargs.get('user_prompt', default_user_prompt.get(dataset_name, ""))
-                    original_prompt = f"Context: {sample.original_context}\nQuestion: {sample.original_question}\n {user_prompt}"
-                    perturbed_prompt = f"Context: {sample.perturbed_context}\nQuestion: {sample.perturbed_question}\n {user_prompt}"
-                    sample.expected_results = model(original_prompt)
-                    sample.actual_results = model(perturbed_prompt)
+                    prompt_template = """Context: {context}\nQuestion: {question}\n """ + user_prompt
+                    # original_prompt = f"Context: {sample.original_context}\nQuestion: {sample.original_question}\n {user_prompt}"
```

Comment:
What was this for?

Reply:
the original_prompt and perturbed_prompt - are we still keeping them in each test module (robustness, bias, etc)?

Reply:
those comments are references.